src/setupWorker/start/createStart.ts

```diff
@@ -124,6 +124,11 @@ If this message still persists after updating, please report an issue: https://g
         return null
       }
 
+      context.keepAliveInterval = setInterval(
+        () => worker.postMessage('PING'),
```

Comment:
What would happen if the worker has terminated/been killed and we keep posting messages to it?
Should we account for that scenario in any way?

Reply:
If we keep posting messages and the user agent stops the service worker, it will reactivate it, which is most likely the behavior that we'd want (assuming the user never calls `worker.stop`. There is no way for us to check if the worker is stopped or not, so the user would have to call `worker.stop()` to stop posting messages. I can't imagine a scenario where anyone would want to keep the worker active but stop the keep alive, as it'd just lead to weird behavior like some of these issues report.